### DISCUSSION TITLE
Can Automa Handle Window Popup Upload? upload file without INPUT element?

### DISCUSSION THREAD

**Discussion by kolampixel** [AutomaApp/automa#1511] [Q&A]
Hi,
I'm trying to create a workflow to upload videos to business.facebook.com but I can't find an INPUT element to upload using the "upload file" block.

![image](https://github.com/AutomaApp/automa/assets/121801559/b304235e-a3f1-41eb-99f7-f689e294ac87)

Have you tried it or do you already have the script?

**Comment by anggun123** [comment]
Up ,,, saya juga lagi  bingung dengan masalah ini
